lib/country_data.rb

```diff
@@ -128,6 +129,10 @@ def popolo
       def statement_count
         json_with_count.last
       end
+
+      def type
+        popolo[:organizations].find { |o| o[:classification] == 'legislature' }[:type] || ''
```

Comment:
The `|| ''` worries me a little here. I don't think we'd want an empty value ending up in `countries.json`. This data _should_ be available for every legislature (or at least will be when that country is being rebuilt, and thus its stanza here is being built), so I think I'd raise an exception in any case where this isn't filled in, so we can catch it early.